app/src/test/java/net/squanchy/support/system/TestCurrentTime.kt

```diff
@@ -0,0 +1,7 @@
+package net.squanchy.support.system
+
+import org.threeten.bp.ZonedDateTime
+
+class TestCurrentTime(private val time: ZonedDateTime) : CurrentTime {
+    override fun currentDateTime() = time
```

Comment:
Nit: whiteline before this

Reply:
01b40a8840a002e913ff5b7ff7c9caa79c3829a5